cypress/support/step-definitions/button-toggle-steps.js

```diff
@@ -68,3 +68,9 @@ Then('Button Toggle is not grouped', () => {
   buttonTogglePreview().eq(SECOND_ELEMENT).should('have.css', 'margin-left', '10px');
   buttonTogglePreview().eq(THIRD_ELEMENT).should('have.css', 'margin-left', '10px');
 });
+
+When('I click on Button Toggle', (element) => {
```

Comment:
`element` is never propagated from the step

Reply:
Amended to ad a button `{int}`, amending `element` to `button` variable 👍